Help/MAC_Hints.rst

```diff
@@ -13,7 +13,15 @@ Coverage Installation Hints for macOS Users:
     $ brew install python3
     $ python3 --version   # To check the installed version
 
-3. Create Virtual Environments with pyvenv
+3. Install pyenv with homebrew
+------------------------------
+
+::
+
+    $ brew update
+    $ brew install pyenv
+
+4. Create Virtual Environments with pyvenv
```

Comment:
No they do not? `pyvenv` is a deprecated script wrapper for `python -m venv` IIRC. Nothing to do with pyenv.